tools/scripts/utils.ts

```diff
@@ -0,0 +1,158 @@
+import {
+  ChildProcess,
+  SpawnOptions,
+  execSync,
+  spawn,
+} from 'node:child_process';
+import { RegistryData, RegistryResult } from './types';
+
+export function configureRegistry({
+  host,
+  registry,
+  registryNoProtocol,
+}: RegistryData) {
+  /**
+   * TODO
+   */
+  // npm
+  process.env.npm_config_registry = registry;
+  // yarnv1
+  process.env.YARN_REGISTRY = registry;
+  // yarnv2
+  process.env.YARN_NPM_REGISTRY_SERVER = registry;
+  console.info(`Set NPM and yarn registry process.env`);
+  /**
+   * TODO
+   */
+  process.env.YARN_UNSAFE_HTTP_WHITELIST = host;
+  console.info(`Set yarn whitelíst process.env`);
```

Comment:
Could you add some more info to the TODO comments? Otherwise it's unclear what is missing exactly.

Reply:
done